onnxruntime/python/tools/quantization/quantize.py

```diff
@@ -194,6 +194,10 @@ def quantize_static(model_input,
                                               inserts both QuantizeLinear/DeQuantizeLinear nodes to weight.
             OpTypesToExcludeOutputQuantizatioin = list of op type : Default is []. If any op type is specified, it won't quantize  
                                                                     the output of ops with this specific op types.
+            DedicatedQDQPair = True/False : Default is False. When inserting QDQ pair, multiple nodes can share a single QDQ pair as their inputs.
+                                            If True, it will create identical and dedicated QDQ pair for each node.  
+            OpTypesSupportPerChannelQuantization = list of op type : Default is []. List of op types that has per channel quantization support.
+            QDQChannelAxis = Integer : Default is 0. Channel axis for QDQ pair when per_channel is True.
```

Comment:
>QDQChannelAxis [](http://example.com/codeflow?start=12&length=14)

i don't think all the operators can share same per-channel axis.

Reply:
which operators in TRT supports per-channel?

Reply:
In TRT pretty much we only use per-channel for weights. All activations are using per-tensor.